social-api/src/main/java/org/geekbeacon/social/www/SocialController.java

```diff
@@ -57,6 +58,12 @@ public AuthResponse authorize(@PathVariable("type") SocialType type, HttpServlet
         return null;
     }
 
+    @GetMapping("v1.0/social/{type}/activity")
```

Comment:
I don't know how swagger generates its documentation, but shouldn't there be documentation on the endpoints as well?

Reply:
Yes, there should be. I'll do another  pass to  add more doc  in general, but  Resources and Models should have more doc.